packages/blocks/src/api/validation/index.js

```diff
@@ -548,7 +548,7 @@ export const isEqualTokensOfType = {
  *
  * @param {Object[]} tokens Set of tokens to search.
  *
- * @return {Object} Next non-whitespace token.
+ * @return {Object | void} Next non-whitespace token.
```

Comment:
What's the difference between "void" and undefined. Should we just use "undefined" everywhere?

Reply:
I used `undefined` in all the cases where the function uses an empty `return`

```js
function () {

    if ( true ) {
        return ;
    }
    return 'value'
}
```

And `void` in all cases where the function didn't return in some cases:

```js
function (condition) {
    if ( condition ) {
        return 'value';
    }
}
```

The goal was to differentiate between functions that do still explicitly return and functions that don't

Reply:
I'm not opposed to `undefined` everywhere though. I can see both sides of the argument

Reply:
yeah I personally feel like it should be "undefined" everywhere because that's the actual value returned by JS (especially if the function can return another type in some other condition)

Reply:
I will update this before I merge it 👍 Thanks for your review ❤️